pkg/integration/tests/patch_building/copy_patch_to_clipboard.go

```diff
@@ -41,13 +41,14 @@ var BuildPatchAndCopyToClipboard = NewIntegrationTest(NewIntegrationTestArgs{
 			).
 			PressPrimaryAction().Press(keys.Universal.CreatePatchOptionsMenu)
 
-		t.ExpectPopup().Menu().Title(Equals("Patch Options")).Select(Contains("Copy patch to clipboard")).Confirm()
+		t.ExpectPopup().Menu().Title(Equals("Patch Options")).Select(Contains("copy patch to clipboard")).Confirm()
+
+		t.Wait(1000)
```

Comment:
Do we need a wait here? I think it's synchronous.

Reply:
Yes with out wait its failing some times. Thats why added it

Reply:
Odd, but alright, I don't mind. A second should be enough.